templates/training_job/sdk_update_post_set_output.go.tpl

```diff
@@ -1,3 +1,8 @@
+observed, err := rm.sdkFind(ctx, latest)
+if err != nil {
+    return observed, err
+}
+desired.SetStatus(observed)
```

Comment:
this to get updated status on 1st update loop?

Reply:
It gets the latest status after the update api has been called.